crates/rwasm/prover/src/rwasmtest/call.rs

```diff
@@ -0,0 +1,32 @@
+use crate::rwasmtest::run_rwasm_prover;
+use rwasm_executor::{Opcode, Program};
+
+fn build_rwasm_call_indirect() -> Program {
+    let ops = vec![
+        Opcode::I32Const(0.into()),
+        Opcode::I32Const(2.into()),
+        Opcode::TableGrow(0),
+        Opcode::I32Const(0.into()),
+        Opcode::I32Const(0.into()),
+        Opcode::I32Const(2.into()),
+        Opcode::TableInit(0),
+        Opcode::TableGet(0),
+        Opcode::I32Const(1.into()),
+        Opcode::CallIndirect(0u32),
+        Opcode::TableGet(0),
+        Opcode::Return,
+        Opcode::I32Const(99.into()),
+        Opcode::I32Const(98.into()),
+        Opcode::I32Add,
```

Comment:
could you please write a test that will take params from stack before CallIndirect, and maybe with table not eq to 0, becasue it is default value and some times it can fail with another